scripts/create_hdf5.py

```diff
@@ -86,20 +86,20 @@ def create_hdf5(out, root_dir, short_size):
         if not os.path.isdir(d):
             continue
 
-        for root, _, fnames in sorted(os.walk(d)):
-            if len(fnames) == 0:
-                continue
-
-            num_ex = len(os.listdir(root))
-            print(root, num_ex)
+        for root in os.scandir(d):
+            if root.is_dir():
```

Comment:
should this be `assert root.is_dir()`? Seems like if that's not true then line 91 will fail.

Reply:
Just seems odd that `fnames` is not set if this branch isn't hit.